src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\Xml\Core\XmlSubtreeReader.cs" />
     <Compile Include="System\Xml\Core\XmlSubtreeReaderAsync.cs" />
     <Compile Include="System\Xml\Core\XmlTextEncoder.cs" />
-    <Compile Include="System\Xml\Core\BinaryCompatibility.cs" />
```

Comment:
ah yes it doesn't have an explicit ref. We have another work item tracking how to fix those, for now keep the ProjectReference. Thanks for trying.